api/sql/schema/750$rule.decision.fetch.sql

```diff
@@ -112,7 +118,8 @@ BEGIN
         @amountWeekly = ISNULL(c.amountWeekly, 0),
         @countWeekly = ISNULL(c.countWeekly, 0),
         @amountMonthly = ISNULL(c.amountMonthly, 0),
-        @countMonthly = ISNULL(c.countMonthly, 0)
+        @countMonthly = ISNULL(c.countMonthly, 0),
+        @isMaskChecked = l.isMaskChecked
```

Comment:
Here we need these changes: 

1. `@credentialsCheck = COALESCE(@credentialsCheck, l.credentials, 0)` - i.e. if credentials to be checked were passed outside, we will not care what is in the table.
2. `@maxAmount = ISNULL(@maxAmountParam, l.maxAmount)` - i.e. if maxAmount was passed outside, we will not care about what is in the table.
3. Add conditions in the where clause, that compare `@amount` and `@amount + @amountDaily`, the credentials mask and everything else to the limits.
4. Add in ORDER BY l.priority after c.priority

So this query should find the first row (if any), where any limit is violated. Then depending on `@isTransactionValidate` we should either return the data from that row or raise appropriate error. If limit violation is due to missing/invalid credentials, then the 'exceed' word in the error code error code can be changed to 'unauthorized', for example we have to raise error `rule.unauthorizedMaxLimitAmount` instead of `rule.exceedMaxLimitAmount`